### PR TITLE
Fix  CVE-2020-9488 vulnerability.

### PR DESCRIPTION
**Pull request by radoslaw-orlowski-amido**
### Change description ###

- Fix  CVE-2020-9488
- upgrade Gradle to 6.3 
- fix deprecated Gradle elements 
- upgrade Spring Boot to 2.2.6.RELEASE 
- upgrade idamBob to 2.0.1
- update to chart-java 2.18.0 

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -31,19 +31,24 @@ allprojects  {
   sourceCompatibility = 1.8
   targetCompatibility = 1.8
 
-  def idamBomVersion = '1.9.7'
+  def idamBomVersion = '2.0.1'
```
Comment (nikola-naydenov-hmcts) [inline build.gradle:34]: can we not make most of these changes in the bom directly?

File: charts/idam-web-public/values.aat.template.yaml [hunk 2]
```diff
@@ -5,6 +5,5 @@ java:
   ingressIP: ${INGRESS_IP}
   consulIP: ${CONSUL_LB_IP}
   replicas: 1
-  aadIdentityName: idam
```
Comment (dougbw) [inline charts/idam-web-public/values.aat.template.yaml:8]: we need this

File: build.gradle (hunk 1 above)
Comment (radoslaw-orlowski-amido) [reply build.gradle:34]: If we control this repo, then maybe. But I'd rather do it properly, it seems that it's used in all web-apps, that would require a separate issue IMO so all affected apps can be tested.

**Comment by radoslaw-orlowski-amido** [comment]
@tbamido , @nikola-naydenov-hmcts : https://tools.hmcts.net/jira/browse/SIDM-4181

File: build.gradle (hunk 1 above)
Comment (nikola-naydenov-hmcts) [reply build.gradle:34]: idam bom is ours and fixing it there will allow just bumping the bom in the other apps to fix this

File: build.gradle [hunk 3]
```diff
@@ -31,8 +31,7 @@ allprojects  {
   sourceCompatibility = 1.8
   targetCompatibility = 1.8
 
-  def idamBomVersion = '1.9.7'
-  ext['tomcat.version'] = '9.0.31'
```
Comment (radoslaw-orlowski-amido) [inline build.gradle:35]: no longer needed, fixed in the newer version of SpringBoot

File: build.gradle [hunk 4]
```diff
@@ -31,8 +31,7 @@ allprojects  {
...
+  def idamBomVersion = '2.1.0'
```
Comment (radoslaw-orlowski-amido) [inline build.gradle:34]: ref: https://github.com/hmcts/idam-bom/pull/94

File: build.gradle (hunk 1 above)
Comment (radoslaw-orlowski-amido) [reply build.gradle]: Done.

**Pull request merged**